indico/modules/events/timetable/schemas.py

```diff
@@ -76,3 +76,5 @@ class Meta:
     session_block = fields.Nested(TimezoneAwareSessionBlockSchema)
     session_id = fields.Integer(dump_only=True)
     duration = fields.TimeDelta(required=True)
+    parent_id = fields.Integer(allow_none=True, load_only=True)
+    session_block_id = fields.Integer(allow_none=True, load_only=True)
```

Comment:
I wonder if we can get session_block_id on load dynamically instead of doing `session_block_id = data.pop('timetable_entry', {}).get('parent', {}).get('session_block_id')` in the code before.